imports/api/setup.app-tests.js

```diff
@@ -0,0 +1,51 @@
+import { Meteor } from 'meteor/meteor';
+import { resetDatabase } from "meteor/xolvio:cleaner";
+
+Meteor.methods({
+  "createTestUser"() {
+    resetDatabase();
+
+    Accounts.createUser({
+      email : "kyvik_bcn@yahoo.es",
+      password : "banana"
+    });
+
+    const meteorUser = Meteor.users.findOne();
+
+    Chatter.User.insert({
+      userId: meteorUser._id,
+      nickname: meteorUser.emails[0].address
+    });
+  },
+
+  "createTestAdmin"() {
+    resetDatabase();
+
+    Accounts.createUser({
+      email : "kyvik_bcn@yahoo.es",
+      password : "banana"
+    });
+
+    const meteorUser = Meteor.users.findOne();
+
+    Chatter.User.insert({
+      userId: meteorUser._id,
+      nickname: meteorUser.emails[0].address,
+      userType: "admin"
+    });
+  },
+
+  "createTestRoom"() {
+    Chatter.Room.insert({
+      name: "Test Room",
+      description: "This is the description of the test room"
+    });
+    const room = ChatterRoom.findOne();
```

Comment:
`insert`returns the inserted document's Id, so you don't need to query it again, just save it to a `roomId` variable.